scripts/templates/component/stories/Component.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 import { withReadme } from "storybook-readme";
-import ${Component} from "../${Component}";
+import { ${Component} }from "../../index";
```

Comment:
Missing space before `from` and extra `../`
should be `import { ${Component} } from "../index";`

Reply:
Ah good catch 😞